### PR TITLE
[CHK-1510] fix: remove null-value json properties in node requests

### PR DESCRIPTION
**Pull request by MarcoAGreco**
#### List of Changes

- Remove null-value json properties in node requests
- Add `JsonInclude.Include.NON_NULL` rule to jackson mapper to avoid adding null-value json properties
#### Motivation and Context

- Align to Nodo web-services requirements

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/test/java/it/pagopa/transactions/client/NodeForPspClientTest.java [hunk 1]
```diff
@@ -220,6 +230,51 @@ void shouldReturnOKClosePaymentResponse() {
         assertThat(clientResponse.getOutcome()).isEqualTo(closePaymentResponse.getOutcome());
     }
 
+    @Captor
+    ArgumentCaptor<Publisher<ClosePaymentRequestV2Dto>> bodyCaptor;
+
+    @Test
+    void shouldReturnOKClosePaymentResponseAdditionalInfo() {
```
Comment (pietro-tota) [inline src/test/java/it/pagopa/transactions/client/NodeForPspClientTest.java:237]: I would rather add two tests here on the serialized ClosePaymentRequest string.
One test where additionalPaymentInformations RRN is valued and another one were is not and check that RRN is serialized in the first and not in the second one.

Maybe for let this being testable the used ObjectMapper used to build Webclient can be moved into a producer method inside WebClientsConfig class

File: src/main/java/it/pagopa/transactions/configurations/WebClientsConfig.java [hunk 2]
```diff
@@ -44,6 +45,7 @@ public WebClient nodoWebClient(
             ObjectMapper mapper = new ObjectMapper();
             mapper.registerModule(new JavaTimeModule());
             mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+            mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```
Comment (pietro-tota) [inline src/main/java/it/pagopa/transactions/configurations/WebClientsConfig.java:48]: Remember to apply this fix also into event-dispatcher module for close payment retry requests

File: src/test/java/it/pagopa/transactions/client/NodeForPspClientTest.java (hunk 1 above)
Comment (MarcoAGreco) [reply src/test/java/it/pagopa/transactions/client/NodeForPspClientTest.java:230]: Thanks, done with 88c9bb9 👍

**Review by pietro-tota — APPROVED** [review]
🚀

**Pull request merged**